express/blocks/simplified-pricing-cards/simplified-pricing-cards.js

```diff
@@ -0,0 +1,306 @@
+  try {
+    const elements = pricingArea.querySelectorAll('p');
+    const year2PricingToken = Array.from(elements).find(
+      (p) => p.textContent.includes(YEAR_2_PRICING_TOKEN),
+    );
+    if (!year2PricingToken) return;
+    if (y2p) {
+      year2PricingToken.innerHTML = year2PricingToken.innerHTML.replace(
+        YEAR_2_PRICING_TOKEN,
+        `${y2p} ${priceSuffix}`,
+      );
+    } else {
+      year2PricingToken.textContent = '';
+    }
+  } catch (e) {
+    window.lana.log(e);
+  }
+}
+
+function handlePriceSuffix(priceEl, priceSuffix, priceSuffixTextContent) {
+  const parentP = priceEl.parentElement;
+  if (parentP.children.length > 1) {
+    Array.from(parentP.childNodes).forEach((node) => {
+      if (node === priceEl) return;
+      if (node.nodeName === '#text') {
+        priceSuffix.append(node);
+      } else {
+        priceSuffix.before(node);
+      }
+    });
+  } else {
+    priceSuffix.textContent = priceSuffixTextContent;
+  }
+}
+
+function handleRawPrice(price, basePrice, response) {
+  price.innerHTML = response.formatted;
+  basePrice.innerHTML = response.formattedBP || '';
+  if (basePrice.innerHTML !== '') {
+    price.classList.add('price-active');
+  } else {
+    price.classList.remove('price-active');
+  }
+  if (response.price.length > 6) {
+    price.classList.add('long-price');
+    basePrice.classList.add('long-price');
+  }
+}
+
+async function createPricingSection(
+  pricingArea,
+  ctaGroup,
+) {
+  pricingArea.classList.add('pricing-area');
+  const priceEl = Array.from(pricingArea.querySelectorAll('a')).filter((a) => a.textContent === PRICE_TOKEN)[0];
+  const pricingBtnContainer = pricingArea.querySelector('.action-area');
+
+  if (pricingBtnContainer && priceEl) {
+    const pricingSuffixTextElem = pricingBtnContainer.nextElementSibling;
+    const placeholderArr = pricingSuffixTextElem.textContent?.split(' ');
+
+    const priceRow = createTag('div', { class: 'pricing-row' });
+    const price = createTag('span', { class: 'pricing-price' });
+    const basePrice = createTag('span', { class: 'pricing-base-price' });
+    const priceSuffix = createTag('div', { class: 'pricing-row-suf' });
+    const response = await fetchPlanOnePlans(priceEl?.href);
+
+    const priceSuffixTextContent = await getPriceElementSuffix(
+      placeholderArr,
+      response,
+    );
+
+    handleRawPrice(price, basePrice, response);
+    handlePriceSuffix(priceEl, priceSuffix, priceSuffixTextContent);
+    handleTooltip(pricingArea);
+    handleYear2PricingToken(pricingArea, response.y2p, priceSuffixTextContent);
+
+    priceRow.append(basePrice, price, priceSuffix);
+    pricingArea.prepend(priceRow);
+    priceEl?.parentNode?.remove();
+    pricingSuffixTextElem?.remove();
+    pricingBtnContainer?.remove();
+  }
+
+  ctaGroup.classList.add('card-cta-group');
+  ctaGroup.querySelectorAll('a').forEach((a, i) => {
+    if (a.href.includes('#_button-fill')) {
```

Comment:
Instead of doing this, I'd recommend making the button primary if the card is the first of type.